CHANGES.md

```diff
@@ -23,11 +23,14 @@
   (@tmcgilchrist, #117)
 - Add a benchmark to judge the respective performance of providing a buffer vs
   letting Yojson create an internal (#134, @Leonidas-from-XIV)
+- Add an optional `suf` keyword argument was added to functions that write
+  serialized JSON, thus allowing NDJSON output. Most functions default to not
+  adding any suffix except for `to_file` (#124, @panglesd) and functions
+  writing sequences of values where the default is `\n` (#135,
+  @Lenoidas-from-XIV)
```

Comment:
```suggestion
  @Leonidas-from-XIV)
```

Reply:
Oops. Yeah. I squashed it all into one commit to keep the commit history clean(er) and will merge when CI is done with it. Thanks!